package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "recharts",
+  "name": "dh-recharts",
```

Comment:
In order to merge to upstream, I think these changes need to be reverted 🤔

Reply:
Whoops! Good catch. I was playing around locally. This wasn't supposed to make the PR. Good catch. I will fix it now.

Reply:
My mistake! this wasn't supposed to be in the PR. I'll remove the changes to package. json. Thanks for catching this.

Reply:
@jkklapp Thanks for looking at my code. I've made the suggested changes if you want to have a look again. Thanks!